subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginIntegrationTest.groovy

```diff
@@ -57,7 +59,9 @@ class StudioPluginIntegrationTest extends StudioPluginTestCase {
         ServerConnection connection = server.waitForIncoming(Duration.ofSeconds(10))
 
         when:
-        def invalidateFile = Paths.get(PathManager.getSystemPath(), "projectModelCache", ".invalidate").toFile()
+        def invalidateFile = ApplicationInfo.instance.majorVersion == "2021" && ApplicationInfo.instance.minorVersionMainPart <= "1"
```

Comment:
Probably doesn't matter much, but doing `<=` with a string doesn't sound safe. How about:

```suggestion
        def invalidateFile = ApplicationInfo.instance.majorVersion == "2021" && ApplicationInfo.instance.minorVersionMainPart in ["0", "1"]
```

Reply:
sure, why not